.travis.yml

```diff
@@ -27,6 +27,7 @@ install:
 
 # command to run tests
 script:
+    - pip install black
```

Comment:
Instead of this, change line 25 to install the dev-requirements

Reply:
(the dev requirements are a superset of the requirements so you should only need to reference the one file)

Reply:
Side note: since this version of black isn't tied to the version in dev-requirements, the tests fail (Black makes minor breaking changes to formatting in each version).

Reply:
Ah yes, indeed, thanks !